src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -90,6 +90,7 @@ public sealed partial class QuicStream
         }
     };
     private MsQuicBuffers _sendBuffers = new MsQuicBuffers();
+    private object _sendBuffersLock = new object();
```

Comment:
:cry:

Reply:
Discussed offline; it's either this or make MsQuicBuffers class instead of struct